mining/block_generate.go

```diff
@@ -86,6 +86,7 @@ func (w *DefaultWorker) Generate(ctx context.Context,
 		Proof:           proof,
 		StateRoot:       newStateTreeCid,
 		Ticket:          ticket,
+		Timestamp:       types.Uint64(time.Now().Unix()),
```

Comment:
Isn't the point of the clock module to encapsulate access to `time.Now()`?  Coming later?

Reply:
Oh, I see your "What's next" in the description.  👍